pkg/cmd/core.go

```diff
@@ -0,0 +1,23 @@
+package cmd
+
+import (
+	"github.com/dcos/dcos-cli/api"
+	"github.com/dcos/dcos-core-cli/pkg/cmd/job"
+	"github.com/spf13/cobra"
+)
+
+// NewCORECommand creates the `core` command with all the available subcommands.
+func NewCORECommand(ctx api.Context) *cobra.Command {
```

Comment:
The case should be `NewCoreCommand`, however there is no `core` command (see my comment below). I think something like `NewCommand`, `NewRootCommand` or `NewDCOSCommand` would be more suited .